src/lib.rs

```diff
@@ -73,6 +73,12 @@ pub enum Error {
 /// // Relative paths are not supported
 /// assert_eq!(windows_to_wsl("Program Files (x86)\\Foo\\bar.txt").unwrap_err(), Error::RelativePath);
 /// assert_eq!(windows_to_wsl("..\\foo\\bar.txt").unwrap_err(), Error::RelativePath);
+///
+/// // Windows WSL paths are converted to the root
+/// assert_eq!(windows_to_wsl("\\\\?\\UNC\\wsl.localhost\\distro\\home\\user\\file").unwrap(), "/home/user/file");
+///
+/// // Generic network paths are not supported right now
+/// assert_eq!(windows_to_wsl("..\\foo\\bar.txt").unwrap_err(), Error::RelativePath);
```

Comment:
Shouldn't this be something like this?

```suggestion
/// assert_eq!(windows_to_wsl("\\\\?\\UNC\\other.domain\\distro\\home\\user\\file").unwrap_err(), Error::InvalidPrefix);
```